src/main/webapp/aboutus.html

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright 2019 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Us</title>
+    <link rel="stylesheet" href="/css/main.css">
+  </head>
+  <body onload="addLoginOrLogoutLinkToNavigation();">
```

Comment:
I'm pretty sure this won't work since the JavaScript file isn't included. You could include the JavaScript file if you wanted, but I'd lean towards just removing this callback to simplify this page.